src/current/v24.1/cockroach-start.md

```diff
@@ -223,6 +227,57 @@ Field | Description
+- Set the environment variable `COCKROACH_WAL_FAILOVER=among-stores` before starting the node.
+
+If you enable WAL failover, you must also update your [logging]({% link {{page.version.version}}/logging-overview.md %}) configuration as described below. This is necessary because although WAL failover alone is not enough to give a node time to recover from a [disk stall]({% link {{ page.version.version }}/cluster-setup-troubleshooting.md %}#disk-stalls) without the node crashing or becoming unstable. The disk unavailability will create instability via another vector: [file logging]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files).
+
+Therefore, if you enable WAL failover, you must also update your [logging]({% link {{page.version.version}}/logging-overview.md %}) configuration as follows:
+
+- (**Recommended**) Configure [remote log sinks]({% link {{page.version.version}}/logging-use-cases.md %}#network-logging) that are not correlated with the availability of your cluster's local disks.
+- If you must log to local disks:
+  1. Disable [audit logging]({% link {{ page.version.version }}/sql-audit-logging.md %}). File-based audit logging and the WAL failover feature cannot coexist. File-based audit logging provides guarantees that every log message makes it to disk, otherwise CockroachDB needs to shut down. Because of this, resuming operations in the face of disk unavailability is not compatible with audit logging.
+  1. Enable asynchronous buffering of [`file-groups` log sinks]({% link {{ page.version.version }}/configure-logs.md %}#output-to-files) using the `buffering` configuration option. The `buffering` configuration can be applied to [`file-defaults`]({% link {{ page.version.version }}/configure-logs.md %}#configure-logging-defaults) or individual `file-groups` as needed.
+  1. Set `max-staleness: 1s` and `flush-trigger-size: 256KiB`.
+  1. When `buffering` is enabled, [`buffered-writes`]({% link {{ page.version.version }}/configure-logs.md %}#buffered-writes) must be explicitly disabled as shown below. This is necessary because `buffered-writes` does not provide true async disk access, but rather a small buffer. If the small buffer fills up, it can cause internal routines performing logging operations to hang, which in turn will cause internal routines doing other important work to hang, potentially affecting cluster stability.
+  1. The recommended logging configuration for using file-based logging with WAL failover is as follows:
+
+        ~~~
+        file-defaults:
+          buffered-writes: false
+          buffering:
+            max-staleness: 1s
+            flush-trigger-size: 256KiB
+            max-buffer-size: 50MiB
+        ~~~
+
+##### Disable WAL failover
+
+To disable WAL failover, you must [restart the node]({% link {{ page.version.version }}/node-shutdown.md %}#stop-and-restart-a-node) and pass the [`--wal-failover=disabled`](#flag-wal-failover) flag to `cockroach start`.
```

Comment:
@jbowens can you also set the ENV VAR to `disabled`, as long as there isn't a start flag that overrides it?

Reply:
yeah

Reply:
ah thanks, forgot to add that for the disable step

made it a bulleted list like the enable step to show both options